wear/src/main/java/com/example/wear/MainActivity.kt

```diff
@@ -0,0 +1,79 @@
+package com.simpletools.calculator.wear
+
+import android.content.res.ColorStateList
+import android.graphics.Color
+import android.os.Bundle
+import android.support.v4.view.ViewCompat
+import android.support.wearable.activity.WearableActivity
+import android.view.View
+import com.simplemobiletools.commons.extensions.performHapticFeedback
+import com.simpletools.calculator.commons.extensions.config
+import com.simpletools.calculator.commons.helpers.Calculator
+import com.simpletools.calculator.commons.helpers.CalculatorImpl
+import kotlinx.android.synthetic.main.activity_main.*
+import me.grantland.widget.AutofitHelper
+
+
+class MainActivity : WearableActivity(), Calculator {
+
+    lateinit var calc: CalculatorImpl
+    private var vibrateOnButtonPress = true
+
+    override fun setValue(value: String) {
+        result.text = value
+    }
+
+    override fun setFormula(value: String) {}
+
+    override fun setClear(value: String) {
+        btn_clear.text = value
+    }
+
+    override fun getResult(): String {
+        return result.text.toString()
+    }
+
+    override fun getFormula(): String {
+        return ""
+    }
+
+
```

Comment:
**Style Criticism:** There should be 1 line between functions in Kotlin according to our coding convention.

Reply:
Done

Reply:
Nice!